test/TestProjects/ExactMatchInheritance/ExactMatchInheritance.json

```diff
@@ -349,7 +349,7 @@
                 "name": {
                     "type": "string"
                 },
-                "NEW": {
+                "new": {
```

Comment:
can we reanme this to something else otherwise model 3 won't inherit a property from this type the way it was intended